CHANGELOG.md

```diff
@@ -5,6 +5,11 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
```

Comment:
this is the place to reference changes that have yet to be published...

Reply:
Aha, I see. Sorry I deal with changelogs often but have not had to deal with a "unreleased" change yet.